book/src/warpgrapher/databases.md

```diff
@@ -0,0 +1,54 @@
+# Databases
+
+Warppgrapher supports using either of two database back-ends for graph data:
+
+1. Azure Cosmos DB
+2. Neo4J
+
+Using each of the databases requires correctly selecting a crate feature and 
+setting up appropriate environment variables to allow Warpgrapher to connect 
+with the database.
+
+## Azure Cosmos DB
+
+Add Warpgrapher to your project config:
+
+`cargo.toml`
+
+```toml
+[dependencies]
+warpgrapher = { version = "0.2.0", features = ["cosmos"] }
+```
+
+Then set up environment variables to contact your Cosmos DB:
```

Comment:
Also a good place to mention the partition key requirement.

Reply:
Agreed.  Added.